Makefile

```diff
@@ -21,12 +21,11 @@ PREFIX ?= '/usr/local'
 
 # This doesn't evaluate until it's called. The -D argument is the
 # directory of the target file ($@), kinda like `dirname`.
-MANPAGES ?= $(patsubst %.asciidoc.in,%,$(wildcard ./docs/man/man1/ansible*.1.asciidoc.in))
-ifneq ($(shell which a2x 2>/dev/null),)
-ASCII2MAN = a2x -L -D $(dir $@) -d manpage -f manpage $<
-ASCII2HTMLMAN = a2x -L -D docs/html/man/ -d manpage -f xhtml
+MANPAGES ?= $(patsubst %.rst.in,%,$(wildcard ./docs/man/man1/ansible*.1.rst.in))
+ifneq ($(shell which rst2man 2>/dev/null),)
+ASCII2MAN = rst2man $< $@
 else
```

Comment:
Can we add an extra conditional for `rst2man.py`?

If you just `pip install docutils` you get a file dropped in `bin` with the `.py` extension.

Something like:

```diff
diff --git a/Makefile b/Makefile
index 112a997e7c..734613e05f 100644
--- a/Makefile
+++ b/Makefile
@@ -24,6 +24,8 @@ PREFIX ?= '/usr/local'
 MANPAGES ?= $(patsubst %.rst.in,%,$(wildcard ./docs/man/man1/ansible*.1.rst.in))
 ifneq ($(shell which rst2man 2>/dev/null),)
 ASCII2MAN = rst2man $< $@
+else ifneq ($(shell which rst2man.py 2>/dev/null),)
+ASCII2MAN = rst2man.py $< $@
 else
 ASCII2MAN = @echo "ERROR: rst2man command is not installed but is required to build $(MANPAGES)" && exit 1
 endif
```

Reply:
Thanks for the review @sivel . The problem with that is that if you install it using your package distro you will not get the `.py` extension. Example on Debian:
```
$ dpkg -L python-docutils | grep rst2man
/usr/share/docutils/scripts/python2/rst2man
$ dpkg -L python3-docutils | grep rst2man
/usr/share/docutils/scripts/python3/rst2man
```
So should we specify a fallback with the non-extension version?

Reply:
I was thinking about something like:
```
ifneq ($(shell which rst2man.py 2>/dev/null),)
ASCII2MAN = rst2man.py $< $@
else ifneq ($(shell which rst2man 2>/dev/null),)
ASCII2MAN = rst2man $< $@
else
ASCII2MAN = @echo "ERROR: rst2man from docutils command is not installed but is required to build $(MANPAGES)" && exit 1
endif
```

Reply:
That is what I recommended in my diff; Adding a 2nd branch to check for `.py`.  I just put the `.py` 2nd in my example.  Preferring `rst2man` then `rst2man.py`

Either way.

Reply:
Oh, thanks, sorry I misread :\ . Amended.